content/desktop/contributing-and-collaborating-using-github-desktop/managing-branches.md

```diff
@@ -66,32 +66,6 @@ If you create a branch on {% data variables.product.product_name %}, you'll need
 2. Click **Publish branch**.
   ![The Publish branch button](/assets/images/help/desktop/publish-branch-button.png)
 
-### Switching between branches
```

Comment:
I disagree with the content strategy to move this procedure to the new article. Switching branches is absolutely a way of managing a branch. It used to be the case that you could only stash changes while you were in the process of switching branches, so this article was the only logical place to include this content. While I agree that the new option of right clicking belongs in a new article,  I disagree what this one should be moved there. There are many folks who want to switch branches without stashing and they will never think to look in the "Stashing changes" article for those instructions. Instead, let's mention in the new article:

> You can also stash changes while you're switching branches. For more information, see "Managing branches."

The "retrieving" procedure makes sense to move, but we should link to it here.